src/renderer/actions.ts

```diff
@@ -2395,6 +2378,99 @@ export const OPEN_MESSAGE_BOX = 'OPEN_MESSAGE_BOX';
 export const CLOSE_MESSAGE_BOX = 'CLOSE_MESSAGE_BOX';
 
 
+/**
+ * Bring up File upload/download dialogs
+ */
+
+export function fileUploadInteractive() {
+    return showDialog('fileUploadDialog');
+}
+
+
+export function fileDownloadInteractive() {
+    return showDialog('fileDownloadDialog');
+}
+
+
+/**
+ *
+ * @param dir Directory to be used
+ * @param file File to upload
+ */
+
+export function uploadFiles(dir: string, file: File): ThunkAction {
+    return (dispatch: Dispatch, getState: GetState) => {
+        const state = getState();
+        const webAPIServiceURL = state.communication.webAPIServiceURL;
+
+        selectors.fileAPISelector(state).uploadFiles(dir, file, webAPIServiceURL)
+            .then((res) => {
+                showToast({type: res.status, text: 'Upload finished: ' + res.message});
+            })
+            .catch((error) => {
+                showToast({type: 'error', text: error.toString()});
+                console.error(error);
+            });
+    }
+}
+
+/**
+ *
+ * @param processId Process ID
+ */
+
```

Comment:
Remove action doc, or complete. If latter, remove empty lines between docs and action.